variant_calling_benchmarks/common.py

```diff
@@ -0,0 +1,39 @@
+import pandas
+
+def extract_loci_string(patient, variant_filenames):
+    '''
+    Given a patient and a list of variant CSV files, return a string
+    like "chr1:2-10,chr2-50-51" that gives the loci of all variants
+    containined in the variant files.
+
+    Parameters
+    -----------
+    patient : string
+
+    variant_filesnames : list of string
+
+    Returns
+    -----------
+    string
+
+    '''
+    loci = []
+    for filename in variant_filenames:
+        df = pandas.read_csv(filename)
+        df = df.ix[df.patient == patient]
```

Comment:
Do you mean instead of `.loc`? No good reason, just habit (`.ix` supports both label and integer indexing).

If you mean instead of just `df[df.patient == patient]`, I think the latter is actually discouraged, since it's supposed to be just for selecting a column, but for historic reasons it will also take a boolean array